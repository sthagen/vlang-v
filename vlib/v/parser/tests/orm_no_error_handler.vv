struct User {
	id int
}

fn main() {
	db := true

	_ := sql db {
		select from User
	}
}
