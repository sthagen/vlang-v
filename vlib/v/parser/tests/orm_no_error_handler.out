vlib/v/parser/tests/orm_no_error_handler.vv:8:7: error: V ORM returns a result, so it should have either an `or {}` block, or `!` at the end
    6 |     db := true
    7 | 
    8 |     _ := sql db {
      |          ~~~~~~~~
    9 |         select from User
   10 |     }
